spark/util/helper_functions/misc_functions.py

```diff
@@ -87,3 +87,27 @@ def helper_random(self, *args):
 @ignite_global
 def helper_randint(self, *args):
     return random.randint(0, args[0])
+
+
+@validate_args([Real, Real, Real, Real])
+@ignite_global
+def helper_bounding_box(self, *args):

```

Comment:
Should negative values be allowed or not?

Reply:
Addressed in latest push